AU2/plugins/custom_plugins/CompetencyPlugin.py

```diff
@@ -139,26 +170,71 @@ def on_hook_respond(self, hook: str, htmlResponse, data) -> List[HTMLComponent]:
         return []
 
     def on_event_request_create(self) -> List[HTMLComponent]:
-        return [
-            Dependency(
-                dependentOn="CorePlugin_assassin_pseudonym",
-                htmlComponents=[
-                    AssassinDependentIntegerEntry(
-                        pseudonym_list_identifier="CorePlugin_assassin_pseudonym",
-                        identifier=self.html_ids["Competency"],
-                        title="Extend competency?",
-                        default={},
-                        global_default=GENERIC_STATE_DATABASE.arb_int_state.get(self.plugin_state["DEFAULT"], DEFAULT_EXTENSION)
-                    )
-                ]
-            )
-        ]
+        if GENERIC_STATE_DATABASE.arb_state.get(self.plugin_state["Auto Competency"], "Manual") != "Full Auto":

```

Comment:
Statements of the form

```
if condition:
    return A
else:
   return B
```

can be simplified to

```
if condition:
    return A
return B
```

In general, you want B to be the longer statement with more nesting.

Reply:
Is there a reason for this? The former seems more readable to me, and surely the performance cost is negligible

Reply:
It doesn't end up being more readable because you put heavily indented code at another indentation level more than it needs to be. It gets worse when your else clause is very long, e.g.

```
if skip_work:
    return True
else:
   very
   long
   work
   follows
```

If you then have several nested If statements in the else that could be simplified, you're adding unnecessary indentation and making it harder to follow control flow

Reply:
(there is no performance cost, it's entirely cosmetic - they almost certainly compile to the same thing in Python bytecode)

Reply:
It seems to be a debated topic on stackoverflow at least. I'll go with your preference though for consistency